src/State/ApiResource/Error.php

```diff
@@ -98,14 +97,7 @@ public function __construct(
     #[Groups(['trace'])]
     public ?array $originalTrace = null;
 
-    #[SerializedName('hydra:title')]
-    #[Groups(['jsonld'])]
-    public function getHydraTitle(): ?string
-    {
-        return $this->title;
-    }
-
-    #[SerializedName('hydra:description')]
+    #[SerializedName('description')]
     #[Groups(['jsonld'])]
     public function getHydraDescription(): ?string
```

Comment:
```suggestion
    public function getDescription(): ?string
```

+ enlever SerializedName